src/database/dao/BookingDataDAO.java

```diff
@@ -8,12 +8,22 @@
 import java.sql.Statement;
 import java.sql.Timestamp;
 import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
 import database.dto.BookingDataDTO;
+import database.dto.PaymentMethodDTO;
 
 public class BookingDataDAO extends MainDAO {
 	
 	private static final String SAVE_IN_BOOKING="INSERT INTO booking(entry_date,departure_date,method_payment,price)"+
 			"VALUES(?,?,?,?)";
+	private static final String SELECT_BOOKING_TABLE = "SELECT id, entry_date, departure_date, price, method_payment FROM booking";
+	
+	private static final String SELECT_BOOKING_TABLE_BY_ID="SELECT id, entry_date, departure_date, price, method_payment FROM booking WHERE id=?";
+	
+	private static final String DELETE_BOOKING_BY_ID="DELETE FROM booking WHERE id=?";
+	
+	private static final String MODIFY_BOOKING="UPDATE booking SET entry_date=?,departure_date=?,price=?,method_payment=? WHERE id=?";
```

Comment:
Please update the database script to say `payment_method` instead of `method_payment`.